{
  "comments": [
    {
      "unresolved": false,
      "key": {
        "uuid": "d8cb5a12_1a5e778a",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 5
      },
      "lineNbr": 0,
      "author": {
        "id": 1120458
      },
      "writtenOn": "2021-04-09T20:17:11Z",
      "side": 1,
      "message": "Several of the tests failing here, I can confirm they pass locally. I believe this was racing w/ LKGB and another fix.",
      "revId": "1a3a8effca802995ece193b7d40cd80abba55798",
      "serverId": "85c56323-6fa9-3386-8a01-6480fb634889"
    }
  ]
}